validator/remote/src/integration-test/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClientTest.java

```diff
@@ -151,48 +148,6 @@ public void getValidators_WhenSuccess_ReturnsResponse() {
     assertThat(result.get()).usingRecursiveComparison().isEqualTo(expected);
   }
 
-  @Test
-  void postValidators_MakesExpectedRequest() throws Exception {
-    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_NO_CONTENT));
-
-    apiClient.postValidators(List.of("1", "0x1234"));
-
-    final RecordedRequest request = mockWebServer.takeRequest();
-    assertThat(request.getMethod()).isEqualTo("POST");
-    assertThat(request.getPath()).contains(ValidatorApiMethod.GET_VALIDATORS.getPath(emptyMap()));
-    assertThat(request.getBody().readUtf8()).isEqualTo("{\"ids\":[\"1\",\"0x1234\"]}");
-  }
-
-  @Test
-  public void postValidators_WhenNoContent_ReturnsEmpty() {
-    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_NO_CONTENT));
-
-    assertThat(apiClient.postValidators(List.of("1"))).isEmpty();
-  }
-
-  @ParameterizedTest
-  @ValueSource(ints = {SC_BAD_REQUEST, SC_NOT_FOUND, SC_METHOD_NOT_ALLOWED})
-  public void postValidators_WhenNotExisting_ThrowsException(final int responseCode) {
-    mockWebServer.enqueue(new MockResponse().setResponseCode(responseCode));
-
-    assertThatThrownBy(() -> apiClient.postValidators(List.of("1")))
-        .isInstanceOf(PostStateValidatorsNotExistingException.class);
-  }
-
-  @Test
-  public void postValidators_WhenSuccess_ReturnsResponse() {
-    final List<ValidatorResponse> expected =
-        List.of(schemaObjects.validatorResponse(), schemaObjects.validatorResponse());
-    final GetStateValidatorsResponse response = new GetStateValidatorsResponse(false, expected);
-
-    mockWebServer.enqueue(new MockResponse().setResponseCode(SC_OK).setBody(asJson(response)));
-
-    Optional<List<ValidatorResponse>> result = apiClient.postValidators(List.of("1", "2"));
-
-    assertThat(result).isPresent();
-    assertThat(result.get()).usingRecursiveComparison().isEqualTo(expected);
-  }
-
```

Comment:
these tests feel like they should be valid in the new world order... any reason they don't get migrated?